students/jsward/session03/strformat_lab.py

```diff
@@ -0,0 +1,58 @@
+# String format lab
+
+from decimal import Decimal
```

Comment:
The Decimal type is handy for some things, but not necessary for this assignment -- string formatting will format regular floating point numbers in decimal for you.